api/src/test/resources/org/openmrs/module/xstream/include/RoleSerializationTest.xml

```diff
@@ -1,23 +1,23 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <dataset>
-  <role role="Data Assistant" description="Clerks who perform data entry." uuid="00eb389f-92b2-102c-adee-6014420f8468"/>
-  <role role="Data Manager" description="User who maintains clinical data stored within the OpenMRS repository." uuid="00eb3992-92b2-102c-adee-6014420f8468"/>
-  
-  <role_role parent_role="Data Assistant" child_role="Data Manager"/>
-  
-  <role_privilege role="Data Assistant" privilege="Add Patients"/>
```

Comment:
Was this simply formatting? It is hard to differentiate between the actual changes and formatting.